src/pkg/packaging/dir.props

```diff
@@ -33,8 +33,11 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <SharedHostBrandName>'Microsoft .NET Core $(ProductionVersion) - Host'</SharedHostBrandName>
-    <HostFxrBrandName>'Microsoft .NET Core $(ProductionVersion) - Host FX Resolver'</HostFxrBrandName>
-    <SharedFrameworkBrandName>'Microsoft .NET Core $(ProductionVersion) - Runtime'</SharedFrameworkBrandName>
+    <SharedHostBrandName>'Microsoft .NET Core $(ProductionVersion) Host'</SharedHostBrandName>
+    <SharedHostBrandName Condition="'$(ReleaseBrandSuffix)'!=''">'Microsoft .NET Core $(ProductionVersion) $(ReleaseBrandSuffix) Host'</SharedHostBrandName>
```

Comment:
Instead of repeating this, can you please define a new variable that carries either the value of ProductionVersion or both ProductionVersion and ReleaseBrandSuffix and use that variable in all the locations. This will also make it simpler to modify the value in a single place.